cg/src/domains/management/subjects/subjects.service.js

```diff
@@ -81,6 +64,23 @@ class SubjectsService {
     };
   }
 
+  async getRequestData(id) {
```

Comment:
why did you move this?

Reply:
I'm sure there was a reason xD
I'm moving it close to where it is being used now